homeassistant/components/alarm_control_panel/services.yaml

```diff
@@ -59,3 +59,6 @@ alarm_trigger:
     code:
       description: An optional code to trigger the alarm control panel with.
       example: "1234"
+    delay:
+      description: An optional delay time to override the state one. In seconds.
```

Comment:
To be a bit more specific. "to override the entry delay for that state. In seconds"